src/kerodon/impl.clj

```diff
@@ -86,12 +86,14 @@
     link
     (not-found "link" selector)))
 
-(defn- field-to-selector [elem]
-  (if-let [id (get-in elem [:attrs :id])]
-    (form-element-by {:id id})
-    (if (= "radio" (get-in elem [:attrs :type]))
-      (form-element-by (select-keys (:attrs elem) [:name :value]))
-      (form-element-by {:name (get-in elem [:attrs :name])}))))
+(defn- field-to-selector [elem & [selector]]
+  (if selector
```

Comment:
You might be right that `:form` is unneeded, I'm not entirely sure.

I think the user can still pass a vector which would allow them to drill down further. Generally My advice in these cases is to add something unique to the HTML to avoid creating an overly brittle test.